DESIGN.md

```diff
@@ -325,18 +325,37 @@ In the addition of the previous rules, an operator for `Flowable`:
 
 ### Creation
 
-Creation of the various types should be exposed through factory methods that provide safe construction.
+Unlike RxJava 1.x, 2.x base classes are abstract and stateless and generally no longer wrap an `OnSubscribe` callback - this saves allocation in assembly time without limiting the expressiveness. Operator methods and standard factories still live as final on the base classes. 
 
-```java
-Flowable.create(SyncGenerator generator)
```

Comment:
@stevegury Was the `SyncGenerator` wording supposed to be a rename of the `SyncOnSubscribe`?